### DISCUSSION TITLE
Syntax highlighting

### DISCUSSION THREAD

**Discussion by sean2gh** [starship/starship#2995] [Q&A] [answered]
The demo gif has this beautiful syntax highlighting and colours even for simple things like passing arguments to built in commands. How do I enable that or set that up for Windows Terminal? Or how do I set my zsh to use starship?

Thanks!

**Comment by chipbuster** [comment, marked as answer]
https://starship.rs/faq/#how-do-i-get-command-completion-as-shown-in-the-demo-gif

For zsh, you probably want `zsh-autosuggestions`, `zsh-completions`, and `fast-syntax-highlighting`. I personally install through [zinit](https://github.com/zdharma/zinit) ever since Antibody became deprecated, but you could also install through plugin managers like [Antigen](https://github.com/zsh-users/antigen) or [Zplug](https://github.com/zplug/zplug)
